### DISCUSSION TITLE
Zoxide PowerShell tab completions?

### DISCUSSION THREAD

**Discussion by charlesroper** [ajeetdsouza/zoxide#343] [Q&A] [answered]
I've just installed Zoxide for PowerShell (7.2.1) and just wondering: are there no tab completions? Thanks!

**Comment by ajeetdsouza** [comment, marked as answer]
Tab completions are currently only available on bash/fish/zsh.

**Reply by charlesroper** [reply]
Thanks for the answer. I've switched to [ZLocation](https://github.com/vors/ZLocation) on PowerShell for now, which does do tab completion and works nicely with fzf via the [PSfzf](https://github.com/kelleyma49/PSFzf) module. It's a good alternative for anyone who misses tabs.

But still on Zoxide everywhere else. :)

**Reply by ajeetdsouza** [reply]
Thanks for the links, I'll check them out. I've created an [issue](https://github.com/ajeetdsouza/zoxide/issues/344) for this, in case you'd like to track it.
